src/imports/otp.js

```diff
@@ -45,15 +57,14 @@ export default new (class {
    * @param {object} options - options used to config the method behavior
    * @returns {Promise<object>}
    */
-  async sendOTP(user: UserRecord, options: any = {}): Promise<object> {
-    const { channel = 'sms' } = options
+  async sendOTP(user: UserRecord): Promise<object> {
     const { mobile } = user
-    const { log, service } = this
-    const payload = { to: mobile, channel }
+    const { log } = this
+    const payload = { recipient: mobile, verify: true }
 
     try {
-      this.assertInitialized()
-      return await service.verifications.create(payload)
+      const result = await this.http.post(this.verifyWorkerUrl, payload)
```

Comment:
you could add response interceptor to always return data instead of the whole response object.
then you could do just `return await this.http.post(...`

Reply:
from a short research I made about response interceptors it seems that the whole response object must be returned by the interceptor

Reply:
why did you suggested that ? actually you're free to do anything with response

https://github.com/axios/axios#interceptors
```javascript
// Do something with response data
    return response;
```

we're using custom interceptors while calling FV API and everything works fine